coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -1993,3 +1993,11 @@ def append(context, node):
 def _abs(context, node):
     inputs = _get_inputs(context, node, expected=1)
     context.add(mb.abs(x=inputs[0], name=node.name))
+
+
+@register_torch_op
+def repeat(context, node):
+    print("repeat op")
+    x = context[node.inputs[0]]
+    reps = context[node.inputs[1]]
+    context.add(mb.tile(x=x, reps=reps, name=node.name))
```

Comment:
@mgharbi In pytorch, `sizes` parameter of repeat can refer to more dimensions than the rank of the input. 

For example, with x.shape = (3,) and  `sizes = [4,2]`, output_shape would be `[4,6]` in PyTorch. 

In CoreML (or MIL), this would be error since len(sizes/reps) <= rank(x).  In order to handle this case, we would need to perform `expand_dims` on the input `x` to prepend dimensions of size 1.

I haven't verified this though. Do you think that you can handle this case in this PR? Thanks.

Reply:
I think we can merge this in and handle this case later on. What do you think @DawerG?

Reply:
I'll have a look at this over the weekend.

Reply:
Right @1duo. Yeah, we can handle it later on. A comment might be helpful for now.